src/pages/costModels/costModel/priceListTable.tsx

```diff
@@ -152,15 +139,11 @@ class PriceListTable extends React.Component<Props, State> {
         />
         <WithPriceListSearch initialFilters={{ primary: 'metrics', metrics: [], measurements: [] }}>
           {({ search, setSearch, onRemove, onSelect, onClearAll }) => {
-            const from = (this.state.pagination.page - 1) * this.state.pagination.perPage;
-            const to = this.state.pagination.page * this.state.pagination.perPage;
```

Comment:
Did you happen to try adding a bunch of rates to see if pagination was working? 

After looking at the code that was removed, it looks that pagination was also a client-side implementation. Thinking we should run a quick test before we remove it.

Reply:
would you rather have a client side pagination vs no pagination?

Reply:
it looks like pagination works client side... with the exception of sorting not including pages you aren't currently viewing

Reply:
Since pagination is functional, we should keep it.

I can see the sort doesn't work quite as expected. Although, I believe we may have a workaround below.

In order for a sort to work with pagination, it needs to be applied before rates are paginated. Thinking we could move the sort function from `rateTable` to `priceListTable`, before we `slice` rates.  Then, add an `onSort` prop to `rateTable` so we can pass table events to `priceListTable`.

Does that sound doable to you?